site/docs/get-started/docker.md

```diff
@@ -13,13 +13,13 @@ is available).
 Pull the ManageIQ docker image:
 
 ```bash
-$ sudo docker pull manageiq/manageiq:latest-darga
+$ sudo docker pull manageiq/manageiq:latest-euwe
```

Comment:
good point, I'll change the tag. thanks @simaishi !

btw I see you've also uploaded euwe-1-rc2 GCE build to Google Storage. I assume you'll be taking care of future releases as well. thanks!